docs/docs/recipes.md

```diff
@@ -94,6 +94,83 @@ There are so many ways to add styles to your website; Gatsby supports almost eve
 - Use the CSS-in-JS library [Styled Components](/docs/styled-components/)
 - Use [CSS Modules](/tutorial/part-two/#css-modules)
 
+### Using Styled Components
+
+#### Requirements
+
+- Install the necessary dependencies for styled-components, including the Gatsby plugin
+
+```shell
```

Comment:
This section actually doesn't need to include the steps to install or setup prerequisites so they don't have to repeat what's explained in a lot of other guides already 🙂 (explained [here](https://www.gatsbyjs.org/contributing/docs-templates/#recipes) a little better)

Just a couple bullet points of what should already be setup is good enough.

Reply:
Okay. Should I mention the packages that are needed to install?

Reply:
Yeah since there are multiple it might be good to mention each of them or say something like:

```
- [Install styled-components and its necessary dependencies](/packages/gatsby-plugin-styled-components/) for use in Gatsby
```

to link to the installation steps. The idea is that the recipes just hit how to use X thing, because installation and setup is covered in README's and getting started places elsewhere.